tempto-kafka/src/main/java/io/trino/tempto/fulfillment/table/kafka/KafkaTableManager.java

```diff
@@ -52,30 +47,22 @@
         implements TableManager<KafkaTableDefinition>
 {
     private final String databaseName;
-    private final QueryExecutor prestoQueryExecutor;
     private final Configuration brokerConfiguration;
-    private final String prestoKafkaCatalog;
 
     @Inject
     public KafkaTableManager(
             @Named("databaseName") String databaseName,
             @Named("broker") Configuration brokerConfiguration,
-            @Named("presto_database_name") String prestoDatabaseName,
```

Comment:
Please remove it from `tempto-examples/src/main/resources/tempto-configuration.yaml`